pkg/workflows/dependency_graph.go

```diff
@@ -0,0 +1,248 @@
+package workflows
```

Comment:
These are parts of models.go + state.go, right? Let me know if you modified anything, otherwise I'm assuming it's just copy-paste.

Reply:
yes, just moved things. it's easy to see by reading the corresponding change in core https://github.com/smartcontractkit/chainlink/pull/13235